packages/node-resolve/src/index.js

```diff
@@ -112,11 +121,8 @@ export function nodeResolve(opts = {}) {
       isRelativeImport = true;
     }
 
-    if (
-      !isRelativeImport &&
-      resolveOnly.length &&
```

Comment:
Just checking have we definitely kept the behaviour where the array is empty the same?

I think before if it was empty we would resolve everything (which imo seems backwards but we should keep it the same).